tap_mssql/__init__.py

```diff
@@ -484,7 +501,7 @@ def get_non_cdc_streams(mssql_conn, catalog, config, state):
             filter(
                 lambda s: s.tap_stream_id == currently_syncing
                 and is_valid_currently_syncing_stream(s, state),
-                streams_with_state,
+                ordered_streams,

```

Comment:
`ordered_streams` - the combination of Streams with and without state is used here because it is possible for a "stream without state" to be currently syncing - i.e. it has not been synced and has no state yet, but it is currently syncing and soon will